trulens/utils/typing.py

```diff
@@ -278,6 +278,11 @@ def nested_map(y: OMNested[C, U],
         for i in range(len(y)):
             out.append(nested_map(y[i], fn, nest - 1))
         return y.__class__(out)
+    if isinstance(y, MAP_CONTAINER_TYPE) and nest > 0:
```

Comment:
nitpick - `MAP_CONTAINER_TYPE` could be substituted with `collections.abc.Mapping` as a generic base class to check against